services/QuillLMS/config/routes.rb

```diff
@@ -456,16 +455,16 @@
       get 'activity_health' => 'rule_feedback_histories#activity_health'
       post 'email_csv_data' => 'session_feedback_histories#email_csv_data'
 
-      resources :activities,              except: [:index, :new, :edit]
-      resources :activity_flags,          only: [:index]
-      resources :activity_sessions,       except: [:index, :new, :edit]
-      resources :feedback_histories,      only: [:index, :show, :create]
-      resources :lessons_tokens,          only: [:create]
+      resources :activities, only: [:create, :show, :update, :destroy]
```

Comment:
Very much appreciate replacement of `except` with `only`.  `except` is an easy way to accidentally add stuff you didn't want.

Reply:
My feeling exactly